cmd/secretcli/secret20.go

```diff
@@ -309,13 +323,95 @@ func s20DepositCmd(cdc *codec.Codec) *cobra.Command {
 	return cmd
 }
 
-func s20Withdraw(cdc *codec.Codec) *cobra.Command {
+func s20Redeem(cdc *codec.Codec) *cobra.Command {
 
 	cmd := &cobra.Command{
-		Use:   "withdraw [contract address or label] [amount]",
-		Short: "Convert your secret token back to SCRT",
+		Use:   "redeem [contract address or label] [amount]",
+		Short: "*EXPERIMENTAL* convert your secret token back to SCRT",
 		Long:  `Convert your secret token back to SCRT. This command will only work if the token supports native currency conversion`,
-		Args:  cobra.MinimumNArgs(2),
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			cliCtx := context.NewCLIContextWithInput(inBuf).WithCodec(cdc)
+
+			contractAddrStr, err := addressFromBechOrLabel(args[0], cliCtx)
+			if err != nil {
+				return err
+			}
+
+			contractAddr, err := sdk.AccAddressFromBech32(contractAddrStr)
+			if err != nil {
+				return errors.New("invalid contract address or label")
+			}
+
+			amount := args[1]
+
+			msg := handleRedeemMsg(amount)
+
+			return cli.ExecuteWithData(cmd, contractAddr, msg, "", false, "", "", cdc)
+		},
+	}
+
+	return cmd
+}
+
+func s20SendCmd(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "send [contract_address or label] [to_account] [amount] [optional: callback_message]",
+		Short: "*EXPERIMENTAL* send tokens to another address. Optionally add a callback message",
+		Long: `Send tokens to another address (contract or not). If 'to_account' is a contract, you can optionally add a callback message to this contract.
+If no callback provided, this is identical to 'transfer'.
+WARNING! The tokens will be transferred even if the callback message fails!`,
```

Comment:
Thought this fails the entire transaction?

Reply:
it does:
https://github.com/enigmampc/secret-secret/pull/16/commits/fe13d987d3f9b259754f7ff1019507907116d937

Reply:
Cool, thanks for verifying @reuvenpo !
Will update the desc.